### DISCUSSION TITLE
excel data to word

### DISCUSSION THREAD

**Discussion by rushikesh-jade** [dolanmiu/docx#1713] [Q&A] [answered]
Hi,

I want to convert excel data into word, so can I achieve this using `docx`.
If this is not possible using `docx`, can anyone suggest which library should I used for `excel to word`

**Comment by dolanmiu** [comment, marked as answer]
You can combine multiple libraries together for this feature

`docx` alone can't do it

Try using:
https://www.npmjs.com/package/xlsx together with `docx`

**Reply by rushikesh-jade** [reply]
I'm using `xlsx` to export JSON into excel, which is working fine, but don't know how excel can be converted into word?
Any suggestion?? 🤔

**Reply by dolanmiu** [reply]
You can use `xlsx` to turn excel into JSON:

https://stackoverflow.com/questions/8238407/how-to-parse-excel-xls-file-in-javascript-html5

And then use `docx` to turn JSON to word
